woocommerce/woocommerce-ios
# Product Variations: polish the loading state

Fixes #1813 

## Changes

Set the ghost content style to Woo style for the table view in Product Variations screen (inspired by @pmusolino's PR https://github.com/woocommerce/woocommerce-ios/pull/2379 😃)

## Testing

- Go to the Products tab
- Tap on a variable product
- Tap on the variations row --> the loading state (ghost content) should look nice similar to the screenshots in Dark and Light mode

## Example screenshots

mode | before | after
-- | -- | --
Dark | ![Simulator Screen Shot - iPhone 11 - 2020-06-04 at 13 21 28](https://user-images.githubusercontent.com/1945542/83718286-58360900-a667-11ea-8980-a4814680a816.png) | ![Simulator Screen Shot - iPhone 11 - 2020-06-04 at 13 22 38](https://user-images.githubusercontent.com/1945542/83718287-5b30f980-a667-11ea-9731-39d41a917114.png)
Light | ![Simulator Screen Shot - iPhone 11 - 2020-06-04 at 13 28 13](https://user-images.githubusercontent.com/1945542/83718292-5cfabd00-a667-11ea-955d-51efff835306.png) | ![Simulator Screen Shot - iPhone 11 - 2020-06-04 at 13 27 12](https://user-images.githubusercontent.com/1945542/83718290-5c622680-a667-11ea-96db-99a3e0e2726d.png)





Update release notes:
